### ISSUE TITLE
How to use aws_lb_target_group_attachment

### ISSUE THREAD

**Issue by bereldhuin** [terraform-aws-modules/terraform-aws-alb#134]
Hi,

I managed to create and ALB with the module.
I created an EC2 resource with a count of 2, and I need to attach them to by ALB with aws_lb_target_group_attachment.

I used target_group_arns output and, similarily, instance_ids output from my own ec2 module but I didn't succeed to do so as terraform (0.12.18) can't use multiple loops (for_each or count) inside a same resource.

Do you have any idea of how to do this ?

Thank you very much,

Fred

**Comment by v0112358** [comment]
I am using Terraform 0.12. Bellow code actually work for me
```
module "vm_demo" {
  source  = "terraform-aws-modules/ec2-instance/aws"
  version = "~> 2.0"
.....
}

module "nlb_demo" {
  source  = "terraform-aws-modules/alb/aws"
  version = "~> 5.0"
....
}


resource "aws_alb_target_group_attachment" "alb_target_demo_gw_80" {
  count            = length(module.vm_demo.id)
  target_group_arn = module.nlb_demo.target_group_arns[0]
  target_id        = module.vm_demo.id[count.index]
  port             = 80
}
```

**Comment by antonbabenko** [comment]
Fixed in #191.

Please use [v5.14.0](https://github.com/terraform-aws-modules/terraform-aws-alb/releases/v5.14.0) that has been just released.
